actors/common/src/main/java/org/sunbird/common/quartz/scheduler/ManageCourseBatchCount.java

```diff
@@ -110,4 +119,122 @@ public void execute(JobExecutionContext ctx) throws JobExecutionException {
 
     return info;
   }
+
+  private void updateCourseBatchMap(
```

Comment:
private function -> only invocation passes both values false?
when are we expecting call to function with true?